cmd/geoipupdate/args.go

```diff
@@ -19,10 +19,15 @@ type Args struct {
 }
 
 func getArgs() *Args {
+	confFileDefault := vars.DefaultConfigFile
```

Comment:
Yeah, `GEOIPUPDATE_CONFIG_FILE=""` would have to be set to avoid reading the config file altogether. To avoid using any of the values of that config file one would have to supply each option as an environment variable or command line flag -- the file would still get read but the values should be overridden.

The only other thing I can think of is convoluted and doesn't really solve it ideally either. Something like create the config using flags first, then environment variables, and then the config file. Each time it would only update fields that haven't yet been updated. When it reaches the time to set config from the config file, it could check if the current config has any fields that are zero values and if not avoid reading the file altogether. But that would mean every optional value would need to be set explicitly, so I don't think thats what we want to do.